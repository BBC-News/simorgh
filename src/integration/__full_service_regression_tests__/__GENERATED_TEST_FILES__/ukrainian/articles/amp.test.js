/**
 * @service ukrainian
 * @pathname /ukrainian/articles/cp4l2mrejvdo
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runCrossPlatformTests from '../../../../pages/articles/crossPlatformTests';
import runAmpTests from '../../../../pages/articles/ampTests';
describe(platform.toUpperCase(), () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runAmpTests();
  });
});
