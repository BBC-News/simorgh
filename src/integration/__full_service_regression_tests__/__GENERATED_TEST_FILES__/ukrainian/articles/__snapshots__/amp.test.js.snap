// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP  A11y I can see the skip to content link 1`] = `"Перейти до змісту"`;

exports[`AMP  AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
}
`;

exports[`AMP  AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP  Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/ukrainian"`;

exports[`AMP  Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ВВС не несе відповідальності за контент інших сайтів. Ознайомтеся з нашими правилами зовнішніх посилань."`;

exports[`AMP  Header I can see the branding 1`] = `"BBC News, Україна"`;

exports[`AMP  I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/340d/test/8f2169c0-31fd-11ea-8512-1f30fd3689d9.jpg"`;

exports[`AMP  I can see an image with a caption 2`] = `"Підпис до фото, Це тестове зображення, авторське право BBC, показує карту Франції. Зображення знаходиться в перших трьох блоках і має цей підпис."`;

exports[`AMP  SEO Canonical link 1`] = `"http://localhost:7080/ukrainian/articles/cp4l2mrejvdo"`;

exports[`AMP  SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP  SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP  SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP  SEO Heading level 1 1`] = `"Це заголовок цієї тестової статті"`;

exports[`AMP  SEO Lang attribute 1`] = `"uk"`;

exports[`AMP  SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png",
      "width": 1024,
    },
    "name": "BBC News Україна",
    "noBylinesPolicy": "https://www.bbc.com/ukrainian/institutional-50170368#authorexpertise",
  },
  "dateModified": "2020-01-08T10:07:49.451Z",
  "datePublished": "2020-01-08T10:07:49.451Z",
  "headline": "Це заголовок цієї тестової статті",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/ukrainian/articles/cp4l2mrejvdo",
    "@type": "WebPage",
    "name": "Це заголовок цієї тестової статті",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png",
      "width": 1024,
    },
    "name": "BBC News Україна",
    "publishingPrinciples": "https://www.bbc.com/ukrainian/institutional-50170368",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png",
  "url": "http://localhost:7080/ukrainian/articles/cp4l2mrejvdo",
}
`;

exports[`AMP  SEO OG description 1`] = `"Це заголовок цієї тестової статті"`;

exports[`AMP  SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png"`;

exports[`AMP  SEO OG image alt 1`] = `"BBC News Україна"`;

exports[`AMP  SEO OG locale 1`] = `"uk-UA"`;

exports[`AMP  SEO OG site name 1`] = `"BBC News Україна"`;

exports[`AMP  SEO OG title 1`] = `"Це заголовок цієї тестової статті - BBC News Україна"`;

exports[`AMP  SEO OG type 1`] = `"article"`;

exports[`AMP  SEO OG url 1`] = `"http://localhost:7080/ukrainian/articles/cp4l2mrejvdo"`;

exports[`AMP  SEO Page title 1`] = `"Це заголовок цієї тестової статті - BBC News Україна"`;

exports[`AMP  SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP  SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP  SEO Twitter creator 1`] = `"@BBC_ua"`;

exports[`AMP  SEO Twitter description 1`] = `"Це заголовок цієї тестової статті"`;

exports[`AMP  SEO Twitter image alt 1`] = `"BBC News Україна"`;

exports[`AMP  SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png"`;

exports[`AMP  SEO Twitter site 1`] = `"@BBC_ua"`;

exports[`AMP  SEO Twitter title 1`] = `"Це заголовок цієї тестової статті - BBC News Україна"`;
